src/ngx_http_lua_kong_module.h

```diff
@@ -10,11 +10,20 @@
 typedef struct {
     STACK_OF(X509)      *upstream_client_certificate_chain;
     EVP_PKEY            *upstream_client_private_key;
+    X509_STORE          *upstream_trusted_store;
+    ngx_uint_t          upstream_ssl_verify;
+    ngx_uint_t          upstream_ssl_verify_depth;
+    unsigned            upstream_ssl_verify_set:1,
+                        upstream_ssl_verify_depth_set:1;
```

Comment:
Style: add `unsigned` before the declaration:

`unsigned upstream_ssl_verify_depth_set:1;`